content/applications/hr/employees/new_employee.rst

```diff
@@ -214,125 +284,129 @@ information for the new employee.
-  menu.
-- :guilabel:`Emergency`: Type in the name and phone number of the employee's emergency contact.
+  name for the new address. To edit the new address, click the :guilabel:`Internal Link` arrow to
+  open the address form. On the address form, enter or edit the necessary details. Some other
+  information in the :guilabel:`Private Contact` section may auto-populate, if the address is
+  already listed in the drop-down menu.
+
+  Enter the employee's :guilabel:`Email` address and :guilabel:`Phone` number in the respective
+  fields. Next, enter the employee's :guilabel:`Bank Account Number` using the drop-down menu. If
+  the bank is not already configured (the typical situation when creating a new employee) enter the
+  bank account number, and click :guilabel:`Create and edit..`. A :guilabel:`Create Bank Account
+  Number` form loads. Fill in the information, then click :guilabel:`Save & Close`.
+
+  Then, select the employee's preferred :guilabel:`Language` from the drop-down menu. Next, enter
+  the :guilabel:`Home-Work Distance` in the field. This field is only necessary if the employee is
+  receiving any type of commuter benefits. Lastly, enter the employee's license plate information in
+  the :guilabel:`Private Car Plate` field.
 - :guilabel:`Education`: Select the highest level of education completed by the employee from the
   :guilabel:`Certificate Level` drop-down menu. Options include :guilabel:`Graduate`,
   :guilabel:`Bachelor`, :guilabel:`Master`, :guilabel:`Doctor`, or :guilabel:`Other`. Type in the
   :guilabel:`Field of Study`, and the name of the :guilabel:`School` in the respective fields.
-- :guilabel:`Citizenship`: This section houses all the information relevant to the citizenship of
-  the employee. Some selections use a drop-down menu, as does the :guilabel:`Nationality (Country)`,
-  :guilabel:`Gender`, and :guilabel:`Country of Birth` sections. The :guilabel:`Date of Birth` uses
-  a calendar module to select the date. First, click on the name of the month, then the year, to
-  access the year ranges. Use the :guilabel:`< (left)` and :guilabel:`> (right)` arrow icons,
-  navigate to the correct year range, and click on the year. Next, click on the month. Last, click
-  on the day to select the date. Type in the information for the :guilabel:`Identification No`,
-  :guilabel:`Passport No`, and :guilabel:`Place of Birth` fields.
-- :guilabel:`Dependant`: If the employee has any children, enter the :guilabel:`Number of Children`
-  in the field.
 - :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.
   Type in the :guilabel:`Visa No` and/or :guilabel:`Work Permit No` in the corresponding fields.
-  Using the calendar module, select the :guilabel:`Visa Expire Date` and/or the :guilabel:`Work
+  Using the calendar selector, select the :guilabel:`Visa Expire Date` and/or the :guilabel:`Work
```

Comment:
This change is not what is in the UI- they are changing it for V17 as a request on a ticket I made, but I want to keep the text accurate- so that is why I chose to leave it "expire" in this doc.